flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarOperatorsTest.scala

```diff
@@ -74,6 +74,103 @@ class ScalarOperatorsTest extends ScalarOperatorsTestBase {
       "true")
   }
 
+  @Test
+  def testShiftLeft(): Unit = {
+    testAllApis(
+      3.shiftLeft(3),
```

Comment:
What happens with negative shift arguments? `3.shiftLeft(-3)`? Ditto for `shiftRight`?